test/test.js

```diff
@@ -131,6 +131,16 @@ describe( 'rollup-plugin-babel', function () {
 		});
 	});
 
+	it( 'works with bablerc: false and transform-decorators-legacy (#188)', function () {
+		return rollup.rollup({
+			entry: 'samples/transform-decorators-legacy/main.js',
+			plugins: [ babelPlugin({
+				babelrc: false,
```

Comment:
@Andarist - I noticed that I had to put `babelrc: false` here an list my specific plugins in the test case.  If I left out `babelrc: false`, the preflight check would not find the `.babelrc` file in the sample directory.  I think this also impacts the test case on lines 127-132 (and likely other test cases).

Reply:
Interesting, could u create an issue linking to this comment? This needs an investigation